cmd/backup-manager/app/backup/manager.go

```diff
@@ -96,6 +96,10 @@ func (bm *Manager) ProcessBackup() error {
 		return fmt.Errorf("no br config in %s", bm)
 	}
 
+	if bm.Mode == string(v1alpha1.BackupModeLog) {
+		return bm.performLogBackup(ctx, backup.DeepCopy())
+	}
+
```

Comment:
put following db connection creation code into a small function to make original code clean, we don't write code for new feature, also do small refactor.

Reply:
ok